### ISSUE TITLE
FEATURE REQUEST: Export data via slug 

### ISSUE THREAD

**Issue by andrewlimaza** [CalderaWP/Caldera-Forms#1170]
The option to export the data using the slug instead of the field name; like i said many of the questions are repeated. I need to import the data into a database and using the first row as the field name is not possible as it's the same text. Having the slug there ALSO would be great!

**Comment by Shelob9** [comment]
@andrewlimaza Can you explain the context here? Do you mean CSV export?

**Comment by andrewlimaza** [comment]
@shelob9 yes, user would like to export to CSV using field slug instead of field name.
